### DISCUSSION TITLE
Access to insiders private repo

### DISCUSSION THREAD

**Discussion by LandonTClipp** [squidfunk/mkdocs-material#5615] [Q&A] [answered]
Hi folks (and @squidfunk of course). I subscribed to an individual sponsorship. The sponsor page says I would receive an invite in a few minutes to the insiders repo, but I haven't received one yet. I wasn't sure if this is supposed to be an automated system or if it's something manually done.

Love the project and I'm very excited to get access to the insiders features for my own personal projects.

**Comment by squidfunk** [comment, marked as answer]
Hey Landon!

Thanks for sponsoring, and we're happy to get you on board! It is an automated system, at least for individual accounts, but GitHub sometimes has problems and doesn't correctly deliver events. Please reach out to sponsors@squidfunk.com for sponsorship-related questions, as mentioned in my sponsors profile – this discussion board is reserved for general support ☺️

**Reply by LandonTClipp** [reply]
@squidfunk sorry to ping on the discussion board, I've emailed a few times the past week with no response. Not sure if my emails are getting through 😢

**Reply by katharinalisalin** [reply]
@LandonTClipp, thanks for checking here again – your emails were mistakenly marked as spam and only reached us now. I just messaged you back, and you should be all set now 😊

**Reply by LandonTClipp** [reply]
Thank you!
